### PR TITLE
add variables for mainnet and password, clear extra_args

### PR DESCRIPTION
**Pull request by etam**
This is a backwards incompatible change.
It depends on #10 (the target branch is not master, to show only relevant changes)

### CODE REVIEW

File: templates/golem.service.j2 [hunk 1]
```diff
@@ -5,7 +5,12 @@ After=network.target hyperdrive.service
 [Service]
 Type=simple
 User={{ golem_source_user }}
-ExecStart={{ golem_source_path }}/venv/bin/python {{ golem_source_path }}/golemapp.py -d {{ golem_source_config_dir }} {{ golem_source_unit_extra_args }}
+ExecStart={{ golem_source_path }}/venv/bin/python {{ golem_source_path }}/golemapp.py \
+    -d {{ golem_source_config_dir }} \
+    {{ golem_source_mainnet | ternary('--mainnet', '') }} \
+    --accept-terms \
+    --password {{ golem_source_password }} \
```
Comment (ktomala) [inline templates/golem.service.j2]: I think this should be in double quotes just in case

**Review by ktomala — CHANGES_REQUESTED** [review]
Small change requested

File: templates/golem.service.j2 (hunk 1 above)
Comment (etam) [reply templates/golem.service.j2]: done

**Review by ktomala — APPROVED** [review]

**Comment by etam** [comment]
fff.... I merged this not into master :/

**Comment by etam** [comment]
I merged this into master

**Pull request merged**